### DISCUSSION TITLE
frpc -f key:port 提示 unkown flag -f

### DISCUSSION THREAD

**Discussion by KevinBrother** [natfrp/wiki#427] [使用问题] [answered]
<img width="774" alt="image" src="https://github.com/natfrp/wiki/assets/29111946/7f40da8a-b0e6-4670-9a44-fad092194daa">

**Comment by fakeboboliu** [comment, marked as answer]
如果您使用有困难，我们建议您阅读并跟随 [启动器 Openwrt 教程](https://doc.natfrp.com/launcher/usage.html#openwrt)。
